app/containers/SongContainer/index.js

```diff
@@ -0,0 +1,250 @@
+import React, { useEffect, memo, useState } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { createStructuredSelector } from 'reselect';
+import { compose } from 'redux';
+import get from 'lodash/get';
+import debounce from 'lodash/debounce';
+import isEmpty from 'lodash/isEmpty';
+import { Card, Skeleton, Input } from 'antd';
+import styled from 'styled-components';
+import { injectIntl } from 'react-intl';
+import { injectSaga } from 'redux-injectors';
+import { selectSongContainer, selectSongsData, selectSongsError, selectQuery } from './selectors';
+import { songContainerCreators } from './reducer';
+import songContainerSaga from './saga';
+import T from '@components/T';
```

Comment:
typically we try to follow this order

- third party imports
- alias imports
- relative imports

Reply:
Got it!